recipes/glob/all/conanfile.py

```diff
@@ -0,0 +1,50 @@
+import os
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class GlobConan(ConanFile):
+    name = "glob"
```

Comment:
```suggestion
    name = "p-ranav-glob"
```

Because glob is too generic and on linux, we can use glob directly from system: https://man7.org/linux/man-pages/man3/glob.3.html

Reply:
Makes sense, I'll rename the package and a directory